crates/sui-protocol-config/src/lib.rs

```diff
@@ -3710,6 +3723,7 @@ impl ProtocolConfig {
                                 max_estimate_us: 1_500_000, // 1.5s
                                 stored_observations_num_included_checkpoints: 10,
                                 stored_observations_limit: 20,
+                                stake_weighted_median_threshold: 3334,
```

Comment:
85 is already on its way out to testnet so i think this would have to wait for 86?